### ISSUE TITLE
crash seg fault attempting landing in n/a systems

### ISSUE THREAD

**Issue by topromulan** [endless-sky/endless-sky#1204]
It seems like possibly any system with no things to land on, like Minkar, Lolami, Mora, were the examples I tried. Tried two separate pilots both crashed when I pressed 'L' in those systems.

Have a nice day

```
Program terminated with signal SIGSEGV, Segmentation fault.
#0  StellarObject::GetPlanet (this=this@entry=0x0)
    at source/StellarObject.cpp:84
#1  0x00000000004162c0 in AI::MovePlayer (this=this@entry=0xa1f7578, ship=..., 
Python Exception <type 'exceptions.ValueError'> Cannot find type const std::list<std::shared_ptr<Ship>, std::allocator<std::shared_ptr<Ship> > >::_Node: 
    player=..., ships=std::list) at source/AI.cpp:1889
Python Exception <type 'exceptions.ValueError'> Cannot find type const std::list<std::shared_ptr<Ship>, std::allocator<std::shared_ptr<Ship> > >::_Node: 
#2  0x00000000004199f6 in AI::Step (this=this@entry=0xa1f7578, 
    ships=std::list, player=...) at source/AI.cpp:258
#3  0x0000000000462315 in Engine::CalculateStep (this=this@entry=0xa1f7570)
    at source/Engine.cpp:776
#4  0x000000000046577b in Engine::ThreadEntryPoint (this=0xa1f7570)
    at source/Engine.cpp:752
#5  0x00007f6bd9529970 in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#6  0x00007f6bd97860a4 in start_thread (arg=0x7f6bc66dc700)
    at pthread_create.c:309
#7  0x00007f6bd8c9987d in clone ()
    at ../sysdeps/unix/sysv/linux/x86_64/clone.S:111
```

**Comment by MarcelineVQ** [comment]
can reproduce here, crash seems to come from commit d34f523632617105221776a99401dba8c6a57c6f

**Comment by endless-sky** [comment]
Fixed in 9f132e8.
